backend/db/migrate/20230502130815_remove_validated_enrollments_unprocessed_messages.rb

```diff
@@ -0,0 +1,21 @@
+class RemoveValidatedEnrollmentsUnprocessedMessages < ActiveRecord::Migration[7.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        execute <<-SQL
+          UPDATE events
+          SET processed_at = '#{Time.zone.now.to_s(:db)}'
+          WHERE id IN (
+            SELECT events.id
+            FROM events
+            JOIN enrollments ON enrollments.id = events.enrollment_id
+            WHERE enrollments.status = 'validated' AND events.name = 'notify' AND processed_at IS NULL
+          )
+        SQL
+      end
+      dir.down do
+        raise ActiveRecord::IrreversibleMigration
```

Comment:
Just wondering, is it really safe to raise an irreversibleMigration and having no reversible migration possible ? 
In case the up case is causing some issue, we could not roll back our data. 
I suggest us to write some test too, I think to unsure that this reversible migration is properly working, in order to prevent others problems ?

https://stackoverflow.com/a/621363/12400017